code/src/App.js

```diff
@@ -1,9 +1,55 @@
-import React from 'react'
+import React, { useState } from 'react';
+import { Gender } from './Gender';
+import { Frequency } from './Frequency';
+import { Actions } from './Actions';
+import { Summary } from './Summary';
+import Suggestions from './Suggestions';
 
 export const App = () => {
+  const [gender, setGender] = useState('');
+  const [frequency, setFrequency] = useState();
+  const [action1, setAction1] = useState('');
+  const [action2, setAction2] = useState('');
```

Comment:
Maybe these could have more describing names? It's hard to know what action 1 is in comparison with action 2